core/src/test/java/org/elasticsearch/indices/memory/IndexingMemoryControllerIT.java

```diff
@@ -109,6 +110,45 @@ public void testIndexBufferSizeUpdateInactiveShard() throws InterruptedException
                             shard1.engine().config().getIndexingBufferSize().bytes() + "]"
             );
         }
+
+        // Make sure we also pushed the tiny indexing buffer down to the underlying IndexWriter:
+        assertEquals(EngineConfig.INACTIVE_SHARD_INDEXING_BUFFER.bytes(), getIWBufferSize("test1"));
+    }
+
+    private long getIWBufferSize(String indexName) {
+        return client().admin().indices().prepareStats(indexName).get().getTotal().getSegments().getIndexWriterMaxMemoryInBytes();
+    }
+
+    @Test
+    public void testIndexBufferSizeTwoShards() throws InterruptedException {
+        createNode(Settings.builder().put(IndexingMemoryController.SHARD_INACTIVE_TIME_SETTING, "100000h",
```

Comment:
> In fact: I don't know why the other test must set 100ms since createNode does it as well ... I'll try to remove it!

OK I was wrong: `createNode` sets `indices.memory.interval` to 100ms, but that test sets `shard_inactive_time` to 100ms ... so I think all is good ... I'll push shortly.